pkg/sqlite/load.go

```diff
@@ -820,7 +820,7 @@ func (s *SQLLoader) updatePackageChannels(tx *sql.Tx, manifest registry.PackageM
 			errs = append(errs, err)
 			break
 		}
-		
+
 		// check replaces
 		replaces, err := channelEntryCSV.GetReplaces()
 		if err != nil {
```

Comment:
I think the `.db-journal` files shouldn't be there because I updated the gitignore to ignore those. You should rebase against master.